sdk/servicebus/service-bus/test/stress-test-track-2/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/service-bus": "file:../../azure-service-bus-7.0.0-preview.8.tgz",
+    "@azure/service-bus": "^7.0.0-preview.8",
```

Comment:
Will this download the package from npm, or is it actually grabbing the local service-bus?

Reply:
npm

Reply:
Gotcha. More of a meta question, but do we plan to run stress tests on the version that hasn't been published yet, or do we only expect to run this against an already published version?

Reply:
Sorry 😁

I'd say, the one that has been published.
Could be both too, at this point - the test code refers to the published one. Though I don't anticipate many changes to the API surface from now on, I don't want to be in sync with the master all the time.

To keep things simple, I'm thinking of using the "published version" here. 
On an ad-hoc basis or as needed, we can update it to the current unpublished or dev package in our VMs while testing.

Reply:
Lets use the published version for now